CRM/Aoservicelisting/Form/ProviderApplicationConfirm.php

```diff
@@ -256,20 +256,6 @@ public function submit($values)
 
         E::createWebsite($staffMember['id'], $values['staff_record_regulator'][$rowNumber]);
 
-        if ($rowNumber == 1) {
```

Comment:
Moved this block of code and outside of the concept that it is only about Row 1 because the first regulated staff member may not be primary contact and there for not have an email address to send to